src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.exception.InvalidPathException;
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static final String OUTPUT_PATH = "src/test/resources/test_output.csv";
+    private static FileWriterService fileWriterService;
+
+    @BeforeClass
+    public static void beforeClass() {
```

Comment:
taking into account, that this is a method, let's make naming as it should be with verb as started word, the same for other test classes